frontend/src/components/maps/ParcelPopupView.tsx

```diff
@@ -27,6 +28,13 @@ export const ParcelPopupView = (props: IParcelDetailProps | null) => {
   const dispatch = useDispatch();
   const [projectRoute, setProjectRoute] = useState('');
 
+  const leaflet = useLeaflet();
+  const defaultZoom = () =>
+    leaflet.map?.flyTo([parcelDetail!.latitude as number, parcelDetail!.longitude as number], 14);
```

Comment:
I forget what `parcelDetail!.latitude` syntax does

Reply:
I think the `!` just ensure that the `parcelDetail` exists and is not undefined